src/gauges.rs

```diff
@@ -51,12 +51,11 @@ pub struct PrometheusGauges {
     pub average_slot_time: Gauge,
     // Connection pool for querying
     client: reqwest::Client,
-    whitelist: Whitelist,
```

Comment:
Let's pass the vote account whitelist in a struct field during the initialisation of every struct that uses it. There is no need to keep on passing the same whitelist as a function argument. Before, we had this when we were recomputing it on every iteration. Now it's effectively constant. So all the reasons to set it in `new` methods.

Reply:
Hmm. I think it's a cleaner and less confusing to pass a `&Whitelist` that's clearly named into every function call as necessary. I had a great deal of confusion working with the `whitelist` in `PrometheusGauges` because I had no idea if the whitelist being passed in as struct construction was a node or vote pubkey whitelist. Performance wise it makes no difference, because the whitelist is small and we never clone.

Reply:
Please can call the field `vote_account_whitelist` to avoid confusion.